pkgs/applications/window-managers/gamescope/default.nix

```diff
@@ -0,0 +1,95 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, pkg-config
+, cmake
+, ninja
+, xorg
+, libdrm
+, vulkan-loader
+, wayland
+, wayland-protocols
+, libxkbcommon
+, libcap
+, SDL2
+, pipewire
+, udev
+, pixman
+, libinput
+, libseat
+, xwayland
+, glslang
+, stb
+, wlroots
+, libliftoff
+, lib
+, makeBinaryWrapper
+}:
+let
+  pname = "gamescope";
+  version = "3.11.33-beta1";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "Plagman";
+    repo = "gamescope";
+    rev = "refs/tags/${version}";
+    hash = "sha256-KAvOmxYSwLA6/zOppDzgI7z8+XCMMEPol1vo/SDuRXE=";
+  };
+
+  patches = [ ./use-pkgconfig.patch ];
+
+  buildInputs = [
+    xorg.libXdamage
+    xorg.libXcomposite
+    xorg.libXrender
+    xorg.libXext
+    xorg.libXxf86vm
+    xorg.libXtst
+    xorg.libXres
+    xorg.libXi
+    libdrm
+    libliftoff
+    vulkan-loader
+    glslang
+    SDL2
+    wayland
+    wayland-protocols
+    wlroots
+    xwayland
+    libseat
+    libinput
+    libxkbcommon
+    udev
+    pixman
+    pipewire
+    libcap
+    stb
+  ];
+
+  nativeBuildInputs = [
+    cmake
```

Comment:
It needs both cmake and meson?

Reply:
and ninja? also nativeBuildInputs are usually sorted before buildInputs

Reply:
Hmm I don't think `cmake` is required actually, must have just have been left over from the original I copied. I made a pass to remove unused `buildInputs` but I neglected the `nativeBuildInputs`.  But yes both meson and ninja are legitimately needed.